x-pack/legacy/plugins/license_management/public/np_ready/application/sections/license_dashboard/start_trial/start_trial.js

```diff
@@ -134,7 +132,7 @@ export class StartTrial extends React.PureComponent {
                         authenticationTypeList: 'AD/LDAP, SAML, PKI, SAML/SSO',
                         securityDocumentationLinkText: (
                           <EuiLink
-                            href={securityDocumentationLink}
+                            href={getDocLinks().securityDocumentationLink || ''}
```

Comment:
in what scenario will `getDocLinks().securityDocumentationLink` not exist?

Reply:
Probably only in tests - but definitely safe to remove here!